agent/engine/docker_task_engine.go

```diff
@@ -43,6 +45,10 @@ const (
 	capabilityPrefix             = "com.amazonaws.ecs.capability."
 	capabilityTaskIAMRole        = "task-iam-role"
 	capabilityTaskIAMRoleNetHost = "task-iam-role-network-host"
+	attributePrefix              = "ecs."
+	taskENIAttribute             = "task-eni"
+	taskENIVersion               = "1.0"
```

Comment:
This should be the same as the release version of amazon-ecs-cni-plugins, which is `0.1.0`